data/remnant/remnant ships.txt

```diff
@@ -643,6 +598,51 @@ ship "Petrel"
 
 
 
+ship "Puffin"
+	plural "Puffins"
```

Comment:
No plural needed for "Puffins" since the standard plural form of all ships already adds an "s".
```suggestion
```